.github/workflows/deploy-production.yml

```diff
@@ -15,13 +16,13 @@ jobs:
 
   # up to date scan of the staging instance
   scan-staging:
-    name: Zap scan of the staging site
+    name: Zap Scan
     uses: ./.github/workflows/zap-scan.yml
     with:
       url: "https://fac-staging.app.cloud.gov/"
 
   deploy-infrastructure-production:
-    name: Deploy infrastructure (production)
+    name: Terraform Apply
```

Comment:
For people who don't understand what Terraform is/does, "Deploy infrastructure" may be more accessible and is parallel naming with "Deploy application".

Reply:
updated